src/servicesHost.ts

```diff
@@ -496,12 +505,17 @@ function resolveModuleName(
     // tslint:disable-next-line:no-empty
   } catch (e) {}
 
-  const tsResolution = compiler.resolveModuleName(
-    moduleName,
-    containingFile,
-    compilerOptions,
-    moduleResolutionHost
-  );
+  const tsResolver = (moduleName: string, containingFile: string, compilerOptions: typescript.CompilerOptions, moduleResolutionHost: typescript.ModuleResolutionHost) =>
```

Comment:
Given `tsResolver` is a pure function could we break this out in to a separate function please?  That saves creating a new function each time this runs.